DQMOffline/Trigger/python/TopMonitoring_Client_cff.py

```diff
@@ -3,7 +3,7 @@
 
 
 topEfficiency_elejets = DQMEDHarvester("DQMGenericClient",
-    subDirs        = cms.untracked.vstring("HLT/TopHLTOffline/TopMonitor/Top/EleJet/*"),
+    subDirs        = cms.untracked.vstring("HLT/TopHLTOffline/TopMonitor/EleJet/*"),
```

Comment:
(sorry, I've just realized)
I would suggest to slightly change the structure to 
HLT/Top/EleJet